### ISSUE TITLE
Implement new menu system and sponsorship placeholder

### ISSUE THREAD

**Issue by ntoll** [PyconUK/planning2015#72]
Based on the designs by @hawkz

**Comment by ghickman** [comment]
@ntoll – do we have any other assets beyond this one that was sent [to the list](https://mail.python.org/mailman/private/pyconuk-adm/attachments/20150612/4e155c4b/attachment-0001.jpg)? (not sure if I'm missing something on the list!)

**Comment by inglesp** [comment]
@ghickman -- I think this is the issue we talked about during Monday's meeting, and we agreed that we'd ask @hawkz for help.  Can you update issue this with where we're at now, and then ping @hawkz?

**Comment by ghickman** [comment]
@hawkz – I got as far as getting a working column layout of the menu `<li>`s using the `column-width` and `column-count` properties. I reworked the `header` layout to make the black banner, nav and sponsor block inline-block elements with a wrapper div to constrain their width.

However, where I really got stuck was making this work with the media breakpoints. I've done my work on the [`new-menu-layout](https://github.com/PyconUK/pyconuk.org/tree/new-menu-layout)` branch.
